timemachine/potentials/bonded.py

```diff
@@ -244,3 +245,17 @@ def log_flat_bottom_bond(conf, params, box, bond_idxs, beta):
     # note the extra 1/beta is to be consistent with other potentials
     # so that energies have units of kJ/mol
     return jnp.sum(log_nrgs) / beta
+
+
+def harmonic_positional_restraint(x_init: Array, x_new: Array, box: Array, k: float = 4_000) -> Array:
+    r"""Harmonic positional restraint useful for performing minimization to prevent initial conformations
+    from changing too much.
+
+    This implements a harmonic bond potential, while being PBC aware:
+        V(x_init, x_init, k) = \sum k / 2 * sum((x_new - x_init)^2)
```

Comment:
doc typo
```suggestion
        V(x_new, x_init, k) = \sum k / 2 * sum((x_new - x_init)^2)
```

Reply:
Oops, earlier suggestion might be [backward](https://github.com/proteneer/timemachine/pull/1393/files#diff-bb2d19446b7ea37b639b8b5a06a398d7be930ad81e47d67e143e2da183f07e1cR463):
```suggestion
        V(x_init, x_new, k) = \sum k / 2 * sum((x_new - x_init)^2)
```